vignettes/tech-dm-join.Rmd

```diff
@@ -128,15 +128,10 @@ An example with filtering on a `dm` and then merging:
 
 **Get all flights from Delta Air Lines which didn't depart from John F. Kennedy International Airport in May 2013 - and join all the airports data into the `flights` table.**
 
-Currently, it is important to call `dm_apply_filters()` after piping your conditions.
-Only then the underlying tables and key relations are updated and you can perform a join on the filtered data.
-We are working towards removing this inconvenience [#62](https://github.com/cynkra/dm/issues/62).
-
 ```{r}
 dm_nycflights13() %>%
   dm_filter(airlines, name == "Delta Air Lines Inc.") %>%
   dm_filter(flights, month == 5) %>%
-  dm_apply_filters() %>%
```

Comment:
AFAIK, no longer necessary.

Reply:
This is about to be deprecated.

Reply:
I meant that `dm_join_to_tbl()` will go away. Need to think about how to handle.